mgr/dashboard_v2: Updated reference documentation

LenzGr opened the pull request:
Updated the dashboard chapter in the manual to reflect the changes introduced by Dashboard v2 (e.g. username/password).

Updated description to include a high-level overview of the features available.

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

votdev commented on doc/mgr/dashboard.rst:
```diff
@@ -2,7 +2,29 @@ dashboard plugin
+  are running and which version of Ceph is installed.
+* **Performance counters**: Display detailed statistics for each running service.
+* **Monitors**: List of all MONs, their quorum status, open sessions.
+* **Configuration Reference**: list all available configuration options, their
```
s/list all/List all/

LenzGr replied:
Fixed, thanks!

votdev commented on doc/mgr/dashboard.rst:
```diff
@@ -2,7 +2,29 @@ dashboard plugin
+  in a given pool. 
+* **CephFS**: List all active filesystem clients and associated pools, including
+  their usage statistics.
+* **Object Gateway**: list of all active object gateways and their performance
```
s/list of/List of/

LenzGr replied:
Fixed.

Commit:
mgr/dashboard_v2: Updated reference documentation

Updated the dashboard chapter in the manual to reflect the
changes introduced by Dashboard v2 (e.g. username/password).

Updated description to include a high-level overview of the features
available.

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>
